dotcom-rendering/src/components/SubMeta.tsx

```diff
@@ -184,7 +193,8 @@ export const SubMeta = ({
 						/>
 					</Island>
 					<div css={syndicationButtonOverrides}>
-						{format.design === ArticleDesign.Interactive ? null : (
+						{format.design === ArticleDesign.Interactive ||
+						format.design === ArticleDesign.Gallery ? null : (
```

Comment:
I know this is not something introduced here, but what do you think about moving this check outside the enclosing `<div>`?

Reply:
Just a comment on this component, and maybe other standard page furniture components. Ideally I think we could do away with the need for components like the `<SubMeta>` to know about the context in which it's being used. This invites the component to become unwieldy, as we offer a pattern for making many tiny variations to the design based on context.

In this case, we could pass a prop to flag whether to show / hide the syndication button.

This question needn't block this PR, and perhaps there is a better forum for a discussion like this. I'd be interested to hear your opinions @marjisound and @JamieB-gu, and anyone else who happens to read!